.github/workflows/dlp-pipelines.yml

```diff
@@ -347,8 +347,8 @@ jobs:
         run: |
           inspect_job_data=$(gcloud dataflow jobs list --project ${{env.PROJECT_ID}} --status active --format json --filter="name=${{env.INSPECT_JOB_NAME}}")
           inspect_job_id=$(echo "$inspect_job_data" | jq -r '.[].id')
-          if !$inspect_job_id; then
-            echo "No job found with name: $inspect_job_id."
+          if [[ "$inspect_job_id" == "" ]]; then
```

Comment:
Do we need this quotes around the $inspect_job_id? Also we can use a single = for equality and == is not standard for bash scripting. I suggest checking [this](https://stackoverflow.com/questions/2237080/how-to-compare-strings-in-bash)
Approving but kindly follow the best coding standard as listed in the above link.

Reply:
I have followed the same notation everywhere else in the workflow and it has been tested several times as working successfully. The article also suggests the same. The quotes are optional. There's no strong purpose to remove.